src/Test/Runner.elm

```diff
@@ -1,8 +1,7 @@
 module Test.Runner
     exposing
-        ( Runnable
-        , Runner(..)
-        , run
+        ( Runner
+        , SeededRunners(..)
         , fromTest
         , getFailure
         , isTodo
```

Comment:
I wanted to keep that one separate so runners could always report it.

Seems useful to say, e.g. "the test run failed because you had 2 failures, and PS you also still have 3 todos" as well as "the test run was incomplete because you used `Test.only` but PS you also still have 3 todos"